sklearn/impute/tests/test_impute.py

```diff
@@ -1524,6 +1524,18 @@ def test_iterative_imputer_keep_empty_features(initial_strategy):
     assert_allclose(X_imputed[:, 1], 0)
 
 
+def test_iterative_imputer_constant_fill_value():
+    """Check that we propagate properly the parameter `fill_value`."""
+    X = np.array([[-1, 2, 3, -1], [4, -1, 5, -1], [6, 7, -1, -1], [8, 9, 0, -1]])
+
+    fill_value = 100
+    imputer = IterativeImputer(
+        missing_values=-1, initial_strategy="constant", fill_value=fill_value, max_iter=0
```

Comment:
For the check "scikit-learn.scikit-learn (Linting, Run black)" to pass, it seems that this code must be reformatted as follows. 

```suggestion
        missing_values=-1,
        initial_strategy="constant",
        fill_value=fill_value,
        max_iter=0,
```